src/Vault.sol

```diff
@@ -1051,11 +1252,7 @@ contract Vault is IERC4626, ERC20Permit, ILiquidationSource, IClaimable, Ownable
    * @dev Will revert if the Vault is undercollateralized.
```

Comment:
This dev natspec is no longer correct. Maybe we should change it to something like:
`@dev This function assumes the vault is collateralized. Do not use this function if the vault is undercollateralized!`

Reply:
Ah yes, I've removed it in the following commit since we mention it in external deposit functions calling this one: https://github.com/GenerationSoftware/pt-v5-vault/pull/51/commits/c5c3e043314de1bf1ae8d545f4242e67eee4dec4